lib/ButtonStyle.ts

```diff
@@ -1,4 +1,24 @@
+import Toast from "react-native-root-toast"
+
 export namespace ButtonStyles {
   export const darkColor = "#26282A"
-  export const opacity = "26" // 15% Opacity
+  export const colorOpacity15 = "#26282A26" // 15% Opacity
+  export const colorOpacity50 = "#26282A80" // 50% Opacity
+  export const colorOpacity35 = "#26282A59" // 35% Opacity
 }
+
+export const showToast = (message: string, bottomOffset: number) => {
+  Toast.show(message, {
```

Comment:
Thanks for having this preset here, it will keep the ui consistent. Consider we may have different presets for different kinds of toasts in the future (error, success) Nothing to worry about rn though